src/index.html

```diff
@@ -8,6 +8,101 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+    </div>
```

Comment:
hard to read such formatting
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>
      
      <div class="stars__star"></div>
    </div>
```

Reply:
this one will be better